scrapy_zyte_api/handler.py

```diff
@@ -129,6 +131,8 @@ def _build_client(settings):
                 api_key=settings.get("ZYTE_API_KEY") or None,
                 api_url=settings.get("ZYTE_API_URL") or API_URL,
                 n_conn=settings.getint("CONCURRENT_REQUESTS"),
+                user_agent=settings.get("_ZYTE_API_USER_AGENT")
+                or f"{USER_AGENT} {PYTHON_ZYTE_API_USER_AGENT}",
```

Comment:
nitpick: would it make sense to use `settings.get("_ZYTE_API_USER_AGENT", default=...)` instead?

Reply:
@kmike I see that default for settings.get is None - do you think it worth to set this explicitly here?

Reply:
I mean that instead of `settings.get("_ZYTE_API_USER_AGENT") or ...` you can use `settings.get("_ZYTE_API_USER_AGENT", default=...)` - or is there any issue with that?

Reply:
@kmike ah, got it, right, thanks!

Reply:
The `or` approach usually has the advantage that it lets you use an empty string on the command line to unset the value (e.g. if set at project level), but for this specific setting I guess that’s not much of a concern.